lib/src/endpoints/player.dart

```diff
@@ -185,4 +185,21 @@ class PlayerEndpoint extends _MeEndpointBase {
 
     return retrievePlaybackState ? playbackState() : null;
   }
+
+  /// Transfer playback to a new device and determine if 
+  /// it should start [play]ing. Default is `true`.
+  ///
+  /// The `AuthorizationScope.connect.modifyPlaybackState` needs to be set.
+  /// [retrievePlaybackState] is optional. If `true`, the current playback state
+  /// will be retrieved. Defaults to `true`.
+  Future<PlaybackState?> transfer(String deviceId,
+      [bool play = true, bool retrievePlaybackState = true]) async {
+    assert(deviceId.isNotEmpty, 'No deviceId provided');
+    var jsonBody = jsonEncode({
+      'device_ids': [deviceId],
```

Comment:
Was just about to comment on Spotify accepting an array but here we only send a single device. But then I read the docs real quick and I understand. Once again we are being spotified.